File: UserServer/src/main/java/cc/nobrain/dev/userserver/UserServerApplication.java
package cc.nobrain.dev.userserver;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.domain.EntityScan;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
import org.springframework.cache.annotation.EnableCaching;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication
@EntityScan(basePackages = {"cc.nobrain.dev.userserver"})
@ConfigurationPropertiesScan(basePackages = {"cc.nobrain.dev.userserver"})
@EnableCaching
@EnableScheduling
public class UserServerApplication {

    public static void main(String[] args) {
        SpringApplication.run(UserServerApplication.class, args);
    }

}

File: UserServer/src/main/kotlin/cc/nobrain/dev/userserver/UserServerApplication.kt
package cc.nobrain.dev.userserver

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.domain.EntityScan
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication
import org.springframework.cache.annotation.EnableCaching
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EntityScan(basePackages = ["cc.nobrain.dev.userserver"])
@ConfigurationPropertiesScan(basePackages = ["cc.nobrain.dev.userserver"])
@EnableCaching
@EnableScheduling
class UserServerApplication

fun main(args: Array<String>) {
    runApplication<UserServerApplication>(*args)
}
